src/components/LoginSection.js

```diff
@@ -0,0 +1,45 @@
+import { useContext } from "react";
+import { UserContext } from "../util/UserContext";
+
+export default function LoginSection() {
+  const {
+    fetchGuestList,
+    username,
+    setUsername,
+    isLoggedIn,
+    isChanges,
+    handleLogout,
+    handleUserDataUpdate,
+  } = useContext(UserContext);
+  return (
+    <>
+      {isLoggedIn && <p>Hello {username}!</p>}
+      {isLoggedIn && (
+        <button type="button" onClick={() => handleLogout()}>
```

Comment:
```suggestion
        <button type="button" onClick={handleLogout}>
```
When no parameters are passed on, you can use the shortcut.